datahub-web-react/src/app/ingest/source/builder/RecipeForm/RecipeForm.tsx

```diff
@@ -59,11 +67,38 @@ function getInitialValues(displayRecipe: string, allFields: any[]) {
     return initialValues;
 }
 
-function SectionHeader({ icon, text }: { icon: any; text: string }) {
+function SectionHeader({
+    icon,
+    text,
+    connectionSectionTooltip,
+    filterSectionTooltip,
+    advanceSectionTooltip,
+}: {
+    icon: any;
+    text: string;
+    connectionSectionTooltip?: string;
+    filterSectionTooltip?: string;
+    advanceSectionTooltip?: string;
+}) {
     return (
         <span>
             {icon}
             <HeaderTitle>{text}</HeaderTitle>
+            {connectionSectionTooltip && (
+                <Tooltip placement="top" title={connectionSectionTooltip}>
+                    <HeaderTooltipWrapper />
+                </Tooltip>
+            )}
+            {filterSectionTooltip && (
+                <Tooltip placement="top" title={filterSectionTooltip}>
+                    <HeaderTooltipWrapper />
+                </Tooltip>
+            )}
+            {advanceSectionTooltip && (
+                <Tooltip placement="top" title={advanceSectionTooltip}>
+                    <HeaderTooltipWrapper />
+                </Tooltip>
+            )}
```

Comment:
you can simplify this and just pass in one optional `tooltip` instead of 3 that all render the same thing! then for each instance of this component below you pass in the correct tooltip based on the section

Reply:
Done!